### PR TITLE
Ee 13988/flagging sa only passes in the ui

### PR DESCRIPTION
**Pull request by leighkirky**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "gulp-util": "^3.0.8",
     "mkdirp": "^0.5.1",
     "moment": "^2.22.1",
+    "natives": "^1.1.6",
```
Comment (owainrobertsbjss) [inline package.json]: I can't see where this is used - but we should definitely chat about it!

File: src/app/modules/familymigration/familymigrationResult.js [hunk 2]
```diff
@@ -79,9 +79,18 @@ familymigrationModule.controller('FamilymigrationResultCtrl',
         $scope.outcomeBoxIndividualName = $scope.individual.forename + ' ' + $scope.individual.surname
 
         if (summary.passed) {
-          state = 'passed'
-          $scope.copysummary = $scope.outcomeBoxIndividualName + ' meets the Income Proving requirement'
-          $scope.success = true
+          // Check if Category F for Self-Assessment Income
```
[the remark this answers is not in the document]
Comment (leighkirky) [reply src/app/modules/familymigration/familymigrationResult.js]: Thanks Owain, made the change you suggested above

File: src/app/modules/familymigration/familymigrationResult.js [hunk 3]
```diff
@@ -79,9 +79,18 @@ familymigrationModule.controller('FamilymigrationResultCtrl',
...
+          if (summary.category === 'F') {
+            state = 'passed'
+            $scope.selfEmployment = true
+            $scope.copysummary = 'Check for evidence of current self employment'
+            $scope.assessmentEndDate = FamilymigrationService.calculateEndOfTaxYear(summary.assessmentStartDate)
+            $scope.success = true
+          } else {
+            state = 'passed'
```
[the remark this answers is not in the document]
Comment (leighkirky) [reply src/app/modules/familymigration/familymigrationResult.js]: Changes made

File: package.json (hunk 1 above)
Comment (leighkirky) [reply package.json]: I introduced this after the pages failed to render and it was suggested has a fix on GitHub, which did work However I have removed, deleted the node modules and rebuilt and it's working without.

**Pull request merged**